examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -127,142 +210,243 @@ class _CustomValueIndicatorShape extends SliderComponentShape {
 }
 
 class _SliderDemoState extends State<SliderDemo> {
-  double _value = 25.0;
+  @override
+  Widget build(BuildContext context) {
+    final List<ComponentDemoTabData> demos = <ComponentDemoTabData>[
+      ComponentDemoTabData(
+        tabName: 'SINGLE SLIDER',
```

Comment:
On small devices, this could be truncated. Maybe since slider is already in the appbar title, you can omit it here? Just a thought

Reply:
Good idea, done